lib/quby/answers/services/answer_validator.rb

```diff
@@ -36,10 +36,23 @@ def validate
                   validate_integer(question, validation, value)
                 when :valid_float
                   validate_float(question, validation, value)
+                when :valid_date
+                  value = question.components.each_with_object({}) do |component, hash|
+                    key = question.send("#{component}_key")
+                    hash[component] = answer.send(key)
+                  end
+                  validate_date(question, validation, value)
                 when :regexp
                   validate_regexp(question, validation, value)
                 when :requires_answer
-                  validate_required(question, validation, value)
+                  if question.type == :date
+                    _value = question.answer_keys.each_with_object({}) do |key, hash|
```

Comment:
Waarom hier een _value? Volgens mij kun je value gewoon overschrijven. 
Daarbij heb ik de underscore-variabele in ruby nog nooit gezien buiten ongebruikte parameters.